low-level/solution/lr2/server/health.py

```diff
@@ -242,23 +243,11 @@ def get_cpu_usage(index=2, percpu=False):
             i = i + 1
         return cpu_usage
 
```

Comment:
do we need get_cpu_usage() now still? Also this gives per cpu thread usage ? Move it to os utils class, as we are still keeping other old functions

Reply:
Moved this func to OSUtils class